RocketChat/Rocket.Chat
# [FIX] Create Discussion modal - Parent room input not displaying correct room name

## Proposed changes (including videos or screenshots)


Also removed tick on input (arrow facing down)

Before:
![image](https://user-images.githubusercontent.com/40830821/124507126-82df1d80-dda3-11eb-84bf-a440f84e43fe.png)

After:
![image](https://user-images.githubusercontent.com/40830821/124507070-64792200-dda3-11eb-84d5-a5a7eb93c40f.png)



## Issue(s)


## Steps to test or reproduce

- Enter a DM
- Create a discussion
- Parent room input should be disabled and text should match the room header.
## Further comments
